client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -68,8 +67,8 @@ public boolean isEnableTls() {
 
     public static final class ClientConfigBuilder {
```

Comment:
The javadoc is present here: https://github.com/pravega/pravega/blob/master/client/src/main/java/io/pravega/client/ClientConfig.java#L89. Do you think we should move it to the class header?

Reply:
That javadoc is covering a private method, and it would be better to be exposed via some public class or method. You can leave the current text block prior to `extractCredentials`, but I recommend adding a paragraph before `ClientConfigBuilder` to explain how to use the configuration here.